lib/flatpak-installation.c

```diff
@@ -1147,7 +1147,7 @@ list_remotes_for_configured_remote (FlatpakInstallation  *self,
 }
 
 /**
- * flatpak_installation_list_remotes:
+ * flatpak_installation_list_remotes_by_type:
```

Comment:
@mwleeds Oops. I had to apply this patch manually as it conflicted. Thanks for spotting that.